api/v1/swagger.yaml

```diff
@@ -4000,6 +4016,13 @@ paths:
                 description: >
                   Max 60 characters; case insensitive; allows alpha-numeric characters,
                   underscore (_) and dash (-).
+              sortBy:
```

Comment:
Overall changes look good. I just felt that we can one more line in description on how this sortBy field is used. Something Like "Default empty. When set to some string, the subjects are sorted alphabetically on string value."
@Prashanth-Billa If you want, you can make this change in next PR, I will merge this now